pkg/virt-handler/cache/cache.go

```diff
@@ -46,6 +45,40 @@ import (
 	"kubevirt.io/kubevirt/pkg/watchdog"
 )
 
+type IterableCheckpointManager interface {
+	List() []string
```

Comment:
It wasn't obvious to me that this method returns the list of keys. I'd suggest changing the name or adding a named return parameter. Something like:
```suggestion
	List() (keys []string)
```

Or:
```suggestion
	ListKeys() []string
```